scripts/json2db.pl

```diff
@@ -141,17 +141,23 @@ sub fetch_policies {
     # so can be just be simply stored for reference in policy page.
     my $out_agreements = {};
 
-    foreach my $policy (@{$policies->{policies}}) {
+    foreach my $policy (@$policies) {
         my $policy_id = $policy->{"id"};
         foreach my $agreement (@{$policy->{"agreement_links"}}) {
             my $decision = $agreement->{"decision"};
-            my $chamber = $decision->{"chamber"};
+            my $decision_date = $decision->{"date"};
```

Comment:
why not use the everywhere there is `$decision->{"ref"}`?

Reply:
oh, thinko on my part I meant `$decision->{"date"}` but my brain had already moved on to the ref bit.